server/src/main.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(non_snake_case)]
```

Comment:
This doesn't belong in `main.rs`, it will disable the warning for the entire crate. Should only apply to the JSON structs.